x/staking/keeper/slash.go

```diff
@@ -249,7 +249,7 @@ func (k Keeper) SlashUnbondingDelegation(ctx context.Context, unbondingDelegatio
 			continue
 		}
 
-		if entry.IsMature(now) && !entry.OnHold() {
```

Comment:
Is `entry.OnHold()` deprecated? 

There are still remaining references if your purpose is cleaning up the code